serenata_toolbox/federal_senate/dataset.py

```diff
@@ -27,12 +27,16 @@ def fetch(self):
             except HTTPError as http_error_exception:
                 print('We failed to reach the server')
                 print('Error code ', http_error_exception.reason)
-                print("While fetching, Seranata Toolbox didn't find file: {} \n{}".format(file_path, http_error_exception))
+                print("While fetching, Seranata Toolbox didn't find file: {} \n{}".format(
+                    file_path,
+                    http_error_exception))
                 raise http_error_exception
             except URLError as url_error_exception:
                 print("The server couldn\'t fulfill the request.")
                 print('Reason: ', url_error_exception.reason)
-                print("While fetching, Seranata Toolbox didn't find file: {} \n{}".format(file_path, url_error_exception))
+                print("While fetching, Seranata Toolbox didn't find file: {} \n{}".format(
+                    file_path,
+                    url_error_exception))
```

Comment:
It feels like you missed one space, and we have a matter os style: 

Either… 

```python
print("While fetching, Seranata Toolbox didn't find file: {} \n{}".format(file_path,
                                                                          url_error_exception))
```

…or…

```python
print("While fetching, Seranata Toolbox didn't find file: {} \n{}".format(
    file_path,
    url_error_exception)
)
```